### ISSUE TITLE
@sorted should take branch prefix into account

### ISSUE THREAD

**Issue by zadam** [TriliumNext/Trilium#162]
Currently notes are sorted without regard for prefixes.

**Comment by BeatLink** [comment]
bump

**Comment by rom1dep** [comment]
As far as I can tell, this got added at some point: the branch prefix, when one exists, is currently treated as part of the note title. [Ref in code](https://github.com/TriliumNext/Trilium/blob/b4b5e86a14dcb12f67b1d7ab9a6af1d22d884a5c/apps/server/src/services/tree.ts#L114).

@BeatLink anything you would wish to be different?
@eliandoran this can probably be closed.

**Comment by BeatLink** [comment]
Yes you're right, notes are sorted by prefixes. This issue can be closed
